modules/deployment-scripts/create-kv-certificate/README.md

```diff
@@ -0,0 +1,74 @@
+# Key Vault Certificate Creation
+
+Create Key Vault self-signed certificates. Requires Key Vaults to be using RBAC Authorization, not Access Policies.
+
+## Parameters
+
+| Name                                       | Type     | Required | Description                                                                                                   |
+| :----------------------------------------- | :------: | :------: | :------------------------------------------------------------------------------------------------------------ |
+| `akvName`                                  | `string` | Yes      | The name of the Azure Key Vault                                                                               |
+| `location`                                 | `string` | No       | The location to deploy the resources to                                                                       |
+| `forceUpdateTag`                           | `string` | No       | How the deployment script should be forced to execute                                                         |
+| `rbacRolesNeededOnKV`                      | `string` | No       | The RoleDefinitionId required for the DeploymentScript resource to interact with KeyVault                     |
+| `useExistingManagedIdentity`               | `bool`   | No       | Does the Managed Identity already exists, or should be created                                                |
+| `managedIdentityName`                      | `string` | No       | Name of the Managed Identity resource                                                                         |
+| `existingManagedIdentitySubId`             | `string` | No       | For an existing Managed Identity, the Subscription Id it is located in                                        |
+| `existingManagedIdentityResourceGroupName` | `string` | No       | For an existing Managed Identity, the Resource Group it is located in                                         |
+| `certNames`                                | `array`  | Yes      | An array of Certificate names to create                                                                       |
+| `initialScriptDelay`                       | `string` | No       | A delay before the script import operation starts. Primarily to allow Azure AAD Role Assignments to propagate |
+| `cleanupPreference`                        | `string` | No       | When the script resource is cleaned up                                                                        |
+
+## Outputs
```

Comment:
need to update outputs section - but see comment below first

Reply:
I think what complicates the outputs is the fact that it works with **arrays** of certificates.
I'm starting to wonder if it should just work with a single certificate, and the calling module can do the loops.
@bmoore-msft

Reply:
Oooooo.... I think you might be right.  That simplifies things quite a bit throughout.  Parameters (type safety), the script itself and outputs.  Let's do it.

Reply:
done!